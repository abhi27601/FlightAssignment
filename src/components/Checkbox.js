
import React from 'react';

 const Checkbox = (props) => (

    <label className="col check"> 
         <input name='launch_year' onChange={props.handleCheckChieldElement} type="checkbox" checked={props.isChecked} value={props.value} />
         <span> {props.value}</span>
    </label>

)

export default Checkbox;