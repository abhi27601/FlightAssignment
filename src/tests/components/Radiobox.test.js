import React from 'react';
import { shallow } from "enzyme";
import Checkbox from "../../components/Checkbox";

test('should render expensedahboard page', () => {
    const wrapper = shallow(<Checkbox />)
    expect(wrapper).toMatchSnapshot();
})