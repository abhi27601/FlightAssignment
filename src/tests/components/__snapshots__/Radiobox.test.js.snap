// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expensedahboard page 1`] = `
<label
  className="col check"
>
  <input
    name="launch_year"
    type="checkbox"
  />
  <span>
     
  </span>
</label>
`;
